src/pages/settings/PasswordPage.js

```diff
@@ -192,7 +193,7 @@ class PasswordPage extends Component {
                                             autoCompleteType="password"
                                             textContentType="password"
                                             value={this.state.newPassword}
-                                            hasError={this.state.errors.newPassword || this.state.errors.newPasswordSameAsOld}
```

Comment:
Why did we remove the error check for `newPasswordSameAsOld`?

Reply:
Not we check as a
`hasError={this.state.errors.newPassword || this.props.account.errorType === 'generic'}`

if new password is same as old password at that time: errorCode === 666

Reply:
But this validation should be done on the front end. We should not delegate the validations to the backend when they can be done on the frontend.

Reply:
@parasharrajat I apology to create the confusion. I accidentally miss that. 

PR updated.